tools/cli-wallet/pledge.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/iotaledger/goshimmer/client/wallet"
+)
+
+func execAllowedPledgeNodeIDsCommand(command *flag.FlagSet, cliWallet *wallet.Wallet) {
+	err := command.Parse(os.Args[2:])
+	if err != nil {
+		printUsage(nil, err.Error())
+	}
+
+	fmt.Println()
+	fmt.Println("Querying node about allowed pledge nodeIDs ... ")
+
+	// query about pledgeNodeIDs
+	res, err := cliWallet.AllowedPledgeNodeIDs()
+	if err != nil {
+		panic(err)
+	}
+
+	for key, value := range res {
+		fmt.Printf("\nAllowed %s full nodeIDs:", key.String())
+		fmt.Println()
```

Comment:
You could add `\n` in the `fmt.Printf` instead of adding an empty line with `fmt.Pritln()`